nix/scripts/version.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
-export NIX_VERSION="2.19.3"
-export NIX_PACKAGE="nixVersions.nix_2_19"
+export NIX_VERSION="2.29.0"
+export NIX_PACKAGE="nixVersions.nix_2_29"
```

Comment:
Why are you upgrading Nix? This is a bad idea without proper testing. If anything it should be aligned with CI:
```yaml
nix_version: '2.24.11'
nix_package: 'nixVersions.nix_2_24'
```
https://github.com/status-im/infra-ci/blob/7d41476eb7d3fa4113adf13d29e3d0544b39aa59/ansible/roles/nix-setup/defaults/main.yml#L4-L5